listenbrainz_spark/hdfs/__init__.py

```diff
@@ -62,6 +62,17 @@ def upload_archive(self, tmp_dump_dir, tar, dest_path, schema, callback=None, fo
         if callback is None:
             raise NotImplementedError('Callback to process JSON missing. Aborting...')
 
+        # Delete '/temp'if it exists
+        if utils.path_exists("/temp"):
```

Comment:
the `/temp` dir here should be a CONSTANT up at the top of the file imo, it's used in a bunch of places.

Reply:
Added it